assets/js/vue/cta.js

```diff
@@ -117,6 +118,6 @@ export default {
                :data-trigger="trigger ? trigger : null"
     >
       <span class="cta__text">{{ text }}</span>
-      <icon :icon="iconName" v-if="hasIcon" size=" " />
+      <icon :icon="iconName" v-if="hasIcon" :size="size" />
```

Comment:
Verstehe nicht wieso der Parameter jetzt von Außen in den Button geworfen werden kann. Das ändert ja auch den Default Case nicht und https://c4a8-web.github.io/shared-components/?path=/story/components-services--services ist weiterhin kaputt oder? Du musst nur den defaultCase im CTA setzen und den an Icon übergeben, wenn du dir die History von icon.js ansiehst, dann kannst du feststellen, dass vorher 'medium' default war und jetzt 'large', müsstest hier also mMn. medium übergeben.